### ISSUE TITLE
2 Questions: Lecture 4, 9

### ISSUE THREAD

**Issue by meier-johannes94** [APMLA-2021/APMLA-WS_21-22_material#13]
I have two more questions: 
- Lecture 4: What do we try to find with the self-consistency? Do we want to find m, J, h simultaneously? Is there only one solution? 
- Lecture 9: Equation 17 contains i,j that we don't have? Is this a mistake? What do we have in the denominator?

**Comment by cdebacco** [comment]
- Lecture 4: the self-consistency aims at finding m, the mean (or first moment) of both MF approximation and the true distribution. We are assuming J,h as known, as this is the direct/forward problem (as opposed to the inverse problem where we observe the m and aim at inferring J,h). There could be more than one solution. In fact, often there is a trivial solution (depending on J,h, e.g. when all the variables are all + or - 1).
- Lecture 9 : I do not see i,j in that equation, perhaps is a different equation or different lecture? L9 eq 17 shows >= elbo(q)

**Comment by meier-johannes94** [comment]
@cdebacco 
Thanks! 
It is not lecture 9 but lecture 8. Sorry :-)

**Comment by cdebacco** [comment]
You are right, the i,j indices in the denominator can be neglected after transforming everything into r,s indices. In fact, the denominator disappears anyway when taking derivative (e.g. for MLE or MAP estimate), as it does not depend on the parameters.
Thanks for spotting it!
